library/src/main/java/com/matthewtamlin/avatar/rules/AvatarRule.java

```diff
@@ -335,7 +336,13 @@ public static Builder builder() {
 		private final Statement baseStatement;
 		
 		private Throwable baseThrowable;
-		
+
+		@Override
+		public SourceVersion getSupportedSourceVersion()
+		{
```

Comment:
pls move to the previous line for formatting.

`public SourceVersion getSupportedSourceVersion() {`